app/views/BusinessEntityOther.scala.html

```diff
@@ -89,6 +90,9 @@
 @detailsBlock = {
     @strong{ @messages("businessEntity.charitable-incorporated-organisation") }
     @p{ @messages("businessEntityOther.charitable-incorporated-organisation") }
+    @p{ @messages("businessEntityOther.charitable-incorporated-organisation-2") }
+    @p{ @messages("businessEntityOther.charitable-incorporated-organisation-3") }
+    @p{ @messages("businessEntityOther.charitable-incorporated-organisation-4-start") @link("https://find-and-update.company-information.service.gov.uk", messages("businessEntityOther.charitable-incorporated-organisation-4-link") , isExternal = true )@messages("businessEntityOther.charitable-incorporated-organisation-4-end") }
```

Comment:
you can just add a dot (.) at the end of the link, there is no need to create a message field for a dot.